includes/wc-user-functions.php

```diff
@@ -581,3 +581,55 @@ function wc_disable_author_archives_for_customers() {
 }
 
 add_action( 'template_redirect', 'wc_disable_author_archives_for_customers' );
+
+/**
+ * Hooks into the `profile_update` hook to set the user last updated timestamp
```

Comment:
We should start use periods, since it's a standard for doc blocks/inline docs and WP core uses too.